### ISSUE TITLE
Make Tux name translatable

### ISSUE THREAD

**Issue by jorgesumle** [retux-game/retux#15]
Line 1667 of `retux.py` file:

```
    name = "S.Penguin Tux" if GOD else "Heretic Tux" if HELL else "Tux"
```

Names aren't usually translated, but in Esperanto most people write "[Tukso](https://eo.wikipedia.org/wiki/Tukso)" and "[Linukso](https://eo.wikipedia.org/wiki/Linux)", as the Esperanto alphabet doesn't have the letter x.

Maybe I can make a pull request tomorrow.

**Comment by onpon4** [comment]
Ah, yup, this is an oversight on our part. Also even if "Tux" didn't need a translation (which it does for reasons like the one you stated and also transliteration for languages that don't use Latin script), the "Heretic" and "S.Penguin" parts need to be translated.

🕷️

**Comment by onpon4** [comment]
Fixed with 7a932a2258bf60c416834c6ee6838072e4f7b941 (forgot to note that in the commit message)

🕷️
